composer.json

```diff
@@ -25,10 +25,11 @@
         "laminas/laminas-validator": "^2.15.1"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~1.0.0",
+        "laminas/laminas-coding-standard": "~2.2.1",
         "laminas/laminas-config": "^3.7.0",
         "laminas/laminas-pdf": "dev-master",
-        "phpunit/phpunit": "^9.5.5"
+        "phpunit/phpunit": "^9.3.7",
+        "staabm/annotate-pull-request-from-checkstyle": "^1.7"
```

Comment:
TBH, let's avoid adding this here - makes it for yet another dependency to upgrade, and is not used by our CI automation (where this could/should be contributed instead)